### PR TITLE
Adds off floor active signatures as required

### LINKED ISSUE #103: Implement all active members counting towards 100%
Now that the constitution states that all active members count towards the total number of required signatures, we should update packet and the UI to display that as such.

### PR DESCRIPTION
**Pull request by mxmeinhold**
Resolves #103

### CODE REVIEW

File: packet/commands.py [hunk 1]
```diff
@@ -113,7 +113,8 @@ def create_packets(freshmen_csv):
     print("Fetching data from LDAP...")
     eboard = set(member.uid for member in ldap_get_eboard())
     onfloor = set(member.uid for member in ldap_get_live_onfloor())
-    all_upper = eboard.union(onfloor)
+    offfloor = set(member.uid for member in set(ldap_get_active_members()) - set(ldap_get_live_onfloor()))
+    all_upper = eboard.union(onfloor).union(offfloor)
```
[the remark this answers is not in the document]
Comment (JoelEager) [reply packet/commands.py]: It'd just go with one set of all active members and the existing eboard set. The sig creation loop can then just iterate over the items in the all active members set and use the `in` operator on the eboard set.

File: packet/commands.py [hunk 2]
```diff
@@ -141,26 +142,27 @@ def ldap_sync():
     print("Fetching data from LDAP...")
     eboard = set(member.uid for member in ldap_get_eboard())
     onfloor = set(member.uid for member in ldap_get_live_onfloor())
-    all_upper = eboard.union(onfloor)
+    offfloor = set(member.uid for member in set(ldap_get_active_members()) - set(ldap_get_live_onfloor()))
```
[the remark this answers is not in the document]
Comment (JoelEager) [reply packet/commands.py]: This function especially needs careful testing to make sure all edge cases are still handled correctly.
